fidesctl/setup.py

```diff
@@ -12,6 +12,7 @@
 dev_requires = open("dev-requirements.txt").read().strip().split("\n")
 
 # Human-Readable/Reusable Extras
+# Add these to `optional_requirements.txt` as well
```

Comment:
It took me a second to realize this is a reminder for future dependencies we add, thank you for adding 💙